src/backend/renderer/multigpu/mod.rs

```diff
@@ -1968,8 +2053,8 @@ where
 impl<'a, 'b, R: GraphicsApi, T: GraphicsApi, Target, BlitTarget> Blit<BlitTarget>
     for MultiRenderer<'a, 'b, R, T, Target>
 where
-    <R::Device as ApiDevice>::Renderer: Blit<BlitTarget>,
-    <T::Device as ApiDevice>::Renderer: Blit<BlitTarget>,
+    <R::Device as ApiDevice>::Renderer: Blit<BlitTarget> + Bind<BlitTarget>,
+    <T::Device as ApiDevice>::Renderer: Blit<BlitTarget> + Bind<BlitTarget>,
```

Comment:
Is this not a duplicate trait bound?

Reply:
Nope. Once for the render and once for the target device.

Reply:
Nope. Once for the render and once for the target device.

Reply:
Oh wait I misread Blit as Bind